### DISCUSSION TITLE
Ignore "The" at the beginning of a series or title when sorting

### DISCUSSION THREAD

**Discussion by snowmangoh** [advplyr/audiobookshelf#358] [Ideas]
I would love the option to exclude "The" from being included in the sorting logic if it's at the beginning of a book or series title.

**Comment by advplyr** [comment]
This has been brought up before in discussion but was never put in as an enhancement request. I put it in #361.
I think a lot of people don't realize how new this project is and how valuable it is to put up feature requests or +1 feature requests already there. Thanks for putting in your suggestions!

**Reply by snowmangoh** [reply]
I just didn't understand where to put that... But I gotta tell you... I've been showing your app to all my fellow engineers, and everyone so far has been super impressed by what you've put together in just a few months!

**Comment by advplyr** [comment]
This is implemented in `v1.6.67`. See #361

**Comment by jakob42** [comment]
I'm managing my kids audiobooks with abs (hope that acronym is all right ;)), but we aren't english speaking. Could we get the option to configure the article list to adapt it to other languages?

**Reply by advplyr** [reply]
Abs is great. I think the best way to handle this is to allow you to specify your own prefixes to ignore.

**Reply by advplyr** [reply]
![image](https://user-images.githubusercontent.com/67830747/161158763-9e569424-da54-42ee-a610-67020771c4e2.png)

This update will be included in v2. Specify your own prefixes.

**Reply by advplyr** [reply]
This is included in [v2.0.1](https://github.com/advplyr/audiobookshelf/releases/tag/v2.0.1)
